BossMod/Modules/Shadowbringers/FATE/Formidable.cs

```diff
@@ -218,10 +223,48 @@ class SteamDome : Components.KnockbackFromCastTarget
         public SteamDome() : base(ActionID.MakeSpell(AID.SteamDome), 15) { }
     }
 
-    // TODO: stay/move component
     class DynamicSensoryJammer : Components.CastHint
     {
-        public DynamicSensoryJammer() : base(ActionID.MakeSpell(AID.DynamicSensoryJammer), "Stay still!") { }
+        public DynamicSensoryJammer() : base(ActionID.MakeSpell(AID.DynamicSensoryJammer), "") { }
+        private int ec;
+        private int casting;
```

Comment:
Why is it an int? Should be a bool?

Reply:
why does it matter if i use 0 or 1 or true or false?

Reply:
Because that's the point of boolean types? While you can use integer as a boolean technically, that implies a different meaning (some counter or something). What would a value of 2 mean here etc?

Reply:
ok, I changed it to bool